rust-runtime/aws-smithy-types/src/body/http_body_1_x.rs

```diff
@@ -0,0 +1,269 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+//! Adapters to use http-body 1.0 bodies with SdkBody & ByteStream
+
+use std::pin::Pin;
+use std::task::{ready, Context, Poll};
+
+use bytes::Bytes;
+use http_body_util::BodyExt;
+use pin_project_lite::pin_project;
+
+use crate::body::{Error, SdkBody};
+
+impl SdkBody {
+    /// Construct an `SdkBody` from a type that implements [`http_body_1_0::Body<Data = Bytes>`](http_body_1_0::Body).
+    ///
+    /// _Note: This is only available with `http-body-1-0` enabled._
```

Comment:
I'm not suggesting to remove this (good to be explicit), but will `#![cfg_attr(docsrs, feature(doc_auto_cfg))]` add a similar comment?

Reply:
yeah should remove actually, I just copy-pasted but we'll get a docs attr